forms/gridfield/GridFieldPrintButton.php

```diff
@@ -121,8 +121,10 @@ public function handlePrint($gridField, $request = null) {
 		Requirements::css(FRAMEWORK_DIR . '/client/dist/styles/GridField_print.css');
 
 		if($data = $this->generatePrintData($gridField)){
-			return $data->renderWith("GridField_print");
+			return $data->renderWith(get_class($gridField)."_print");
```

Comment:
Wouldn't this break on subclasses? `renderWith()` itself does not have a concept of a class hierarchy traversal to find "parent class templates", and `get_class()` always retrieves the concrete class of the instance.

How about this? I don't see a clean way to identify the "base class" of `GridField` without hardcoding here.
```
$templates = SSViewer::get_templates_by_class($gridField, '_print', 'GridField);
return $data->renderWith($templates);
```

Happy to merge either way, given that `GridField` is usually not extended directly it's a bit of an edge case.